npc/quests/quests_moscovia.txt

```diff
@@ -8646,9 +8646,10 @@ OnTouch:
 				close;
 			}
 			mes "-You seem to hear the splash but something gleaming raids on you!!-";
+			close2;
 			specialeffect2 EF_FLASHER;
 			percentheal -100,0;
-			close;
+			end;
```

Comment:
maybe `unitkill getcharid(3);` would be better/suitable for killing a char? 
Perhaps would be helpful in future code standardization?